### ISSUE TITLE
More options modal: Missing explicit title for each version

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2022]
## Bug Report

**Problematic behavior**
Each version button has an `aria-label="version item"` attribute. This attribute overrides the text content of each button and is identical for all buttons in the list.

The vocalization will be: "version item, version item, version item…"

**Expected behavior/code**
Each button must have an explicit label so that the user can choose which version they want to restore.

**Steps to Reproduce**
1. Do this...
2. Then this...
3. And then the bug happens!

**Environment**
- Docs version:
- Instance url:

**Possible Solution**
2 possibilities:
- Remove the `aria-label` attribute to allow the text content to be announced,
- Replace the `aria-label` attribute with a clearer label (preferred solution)

```
<button type="button" aria-label="Restaurer la version du 16 mars 2026 à 17:13">
<span>16 mars 2026, 17:13</span>
</button>
```

**Additional context/Screenshots**

<img width="903" height="662" alt="Image" src="https://github.com/user-attachments/assets/3bb6d32f-32c2-4d0b-bc7f-dfc0d6697ecb" />

**Critères RGAA**

**Critère 11.9 :** [[Dans chaque formulaire, l'intitulé de chaque bouton est-il pertinent ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#11.9)](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#11.9)

**Impact** 

A user navigating with their screen reader will not be able to differentiate between each version since the label is the same.
